WalletWasabi.Backend/Global.cs

```diff
@@ -88,11 +89,12 @@ public async Task InitializeAsync(Config config, CoordinatorRoundConfig roundCon
 			var indexFilePath = Path.Combine(indexBuilderServiceDir, $"Index{RpcClient.Network}.dat");
 			var blockNotifier = HostedServices.FirstOrDefault<BlockNotifier>();
 			IndexBuilderService = new IndexBuilderService(RpcClient, blockNotifier, indexFilePath);
-			Coordinator = new Coordinator(RpcClient.Network, blockNotifier, Path.Combine(DataDir, "CcjCoordinator"), RpcClient, roundConfig);
+			Coordinator = new Coordinator(RpcClient.Network, blockNotifier, Path.Combine(DataDir, "CcjCoordinator"), RpcClient, roundConfig,
+				() => _ = serviceProvider.GetRequiredService<SendPushService>().SendNotificationsAsync(roundConfig.IsDebug));
```

Comment:
this func for lazy DI shows that there is a design flaw somewhere. We should fix this later on